api/api.py

```diff
@@ -881,7 +886,10 @@ def local_resource(name: str, cmd: Union[str, List[str]],
                    auto_init: bool=True, serve_cmd: str = "", cmd_bat: str = "",
                    serve_cmd_bat: str = "",
                    allow_parallel: bool=False,
-                   links: Union[str, Link, List[Union[str, Link]]]=[]) -> None:
+                   links: Union[str, Link, List[Union[str, Link]]]=[],
+                   tags: List[str] = [],
```

Comment:
oh whoooops this is supposed to be a secret parameter for now, but I see why you need to mention it in the function signature... it's proooobably fine to mention it here and just not in the list of params below? 🤞

Reply:
Yeah, I can't imagine anyone is using positional args this deep in a `local_resource` definition so it really shouldn't effect anyone either way...I almost omitted it but erred on the side of staying accurate